ouroboros-network/src/Ouroboros/Network/PeerSelection/Governor/Types.hs

```diff
@@ -1381,6 +1383,30 @@ pickPeers PeerSelectionState{localRootPeers, publicRootPeers, knownPeers}
              ++ " which is outside of the set given to pick from"
 
 
+-- | Pick some known peers.
+--
+pickPeers :: (Ord peeraddr, Functor m, HasCallStack)
+          => PeerSelectionState peeraddr peerconn
+          -> PickPolicy peeraddr m
+          -> Set peeraddr -> Int -> m (Set peeraddr)
+{-# INLINE pickPeers #-}
```

Comment:
Why is this needed?

Reply:
Probably not, it's just a tiny wrapper which likely will be inlined anyway.